### ISSUE TITLE
Meaning of quantities displayed during training

### ISSUE THREAD

**Issue by wazizian** [mlelarge/graph_neural_net#7]
I am struggling a bit with the meaning of the quantities displayed during training, which look like this,
```
Epoch: [0][0/625]       Time 3.886 (0.121)      Data 0.092 (0.003)      LR 1.00e-03     Loss 4.2248 (0.1320)    Acc_Max 40.000 (0.025)
```
If I understand well the first value displayed is the last value logged and the second is a temporal average. I have issues with `loss` and `acc_max`.

- The loss is updated with
```
meters['loss'].update(loss.data.item(), n=batch_size)
```
meaning that `loss.data.item()` is supposed to be the sum of `batch_size` individual losses. However, from the [doc of the NLLLoss](https://pytorch.org/docs/stable/nn.html#nllloss), it seems that the loss is already averaged over the batch and so `n=1` would make more sense to me.

- The accuracy is updated with
```
 acc_max, n, bs = eval_score(np_out)
 meters['acc_max'].update(acc_max,n*bs)
```
Here I agree with value of `n*bs` because `acc_max` is the number of right predictions, but then the value displayed is precisely `acc_max` which is therefore not an accuracy. For me, the value displayed should be `acc_max/(n*bs)`.

 - Finally, I was wondering whether 'logger.reset_meters' should be called more often. Otherwise, the displayed average always takes into account the first iterations which can behave badly. Maybe we should reset the meters after each display?

Bests,

Waïss
